components/dashboard/src/admin/WorkspacesSearch.tsx

```diff
@@ -68,27 +68,19 @@ export function WorkspaceSearch(props: Props) {
     const search = async () => {
         setSearching(true);
         try {
-            let searchTerm: string | undefined = queryTerm;
-            const query: AdminGetWorkspacesQuery = {
-                ownerId: props?.user?.id,
-            };
-            if (matchesInstanceIdOrLegacyWorkspaceIdExactly(searchTerm)) {
-                query.instanceIdOrWorkspaceId = searchTerm;
-            } else if (matchesNewWorkspaceIdExactly(searchTerm)) {
-                query.workspaceId = searchTerm;
-            }
-            if (query.workspaceId || query.instanceId || query.instanceIdOrWorkspaceId) {
-                searchTerm = undefined;
+            const query: AdminGetWorkspacesQuery = {};
+            if (matchesInstanceIdOrLegacyWorkspaceIdExactly(queryTerm)) {
+                query.instanceIdOrWorkspaceId = queryTerm;
+            } else if (matchesNewWorkspaceIdExactly(queryTerm)) {
+                query.workspaceId = queryTerm;
```

Comment:
@geropl What was the case before where we would populate `query.instanceId`?

Reply:
@laushinka Not sure I understand your question? IMO we never were able to reliably fill `instanceId` (because `workspaceId`s always looked identical).

Reply:
> Maybe `instanceId` was added with the previous change that introduced this "interpretation" of user input, to pave the way for a explicit choice (dropdown).

Makes sense!